consensus/istanbul/core/core.go

```diff
@@ -230,6 +237,28 @@ func GetAggregatedSeal(seals MessageSet, round *big.Int) (types.IstanbulAggregat
 	return types.IstanbulAggregatedSeal{Bitmap: bitmap, Signature: asig, Round: round}, nil
 }
 
+// GetAggregatedEpochSeal aggregates all the given seals for a the SNARK-friendly epoch encoding
+// to a bls aggregated signature and bitmap
+func GetAggregatedEpochSeal(seals MessageSet, round *big.Int) (types.IstanbulAggregatedEpochSeal, error) {
```

Comment:
What is the round used for here?

In the regular aggregated seal, validators sign the block hash + round. There, it's useful for signatures to be specific to a certain view (round and sequence) for detecting double signing.